website/docs/users-sources/sources/social-logins/google/workspace/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Google Workspace with SAML
+sidebar_label: Workspace SAML
+tags: [integration, saml, google]
+---
+
+<span className="badge badge--primary">Support level: authentik</span>
+
+## What is Google Workspace?
+
+Google Workspace (formerly G Suite) is a collection of cloud computing, productivity and collaboration tools, software and products developed and marketed by Google.
+
+Organizations using Google Workspace allow their users to authenticate into applications using their company email addresses. This guide shows how to set up Security Assertion Markup Language (<abbr>SAML</abbr>) as the authentication method between Google Workspace and authentik.
+
+## SAML Authentication Flow
+
+This sequence diagram shows a high-level flow between user, authentik, Google, and the target application.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant User
+    participant authentik
+    participant Google
+    participant App
+
+    User->>App: User navigates to app...
+    App-->>authentik: ← Redirected to authentik
+    Note over authentik: User chooses "Google Workspace"
+    authentik-->>Google: Redirect to Google →
+    Note over Google: Authenticate with email and password
+    Google-->>authentik: ← Redirect to authentik
+    authentik-->>App: Redirect to app →
+```
+
+In short, the user navigates to the application, is redirected to authentik, chooses Google Workspace as the authentication method, authenticates with Google, and is redirected back to the application.
+
+The key characteristic that makes this an idP-to-idP flow is that Authentik is acting as an intermediary identity provider, brokering trust between your application and Google Workspace.
+
+---
+
+## Integration
+
+By the end of this integration, your authentik instance will allow users to authenticate using their Google Workspace credentials.
```

Comment:
@Teffen oyu and I had a good convo the other day about our need to make it more clear at the beginning of each page what exactly the page is about, what the task/goal of the process described on the page is. This sort of sentence ("By the end of this integration, your authentik instance will allow users to authenticate using their Google Workspace credentials.") is really what we need at very top of page. I'll go back up and make a suggestion.